test/blackbox-tests/test-cases/describe/describe-pp/describe-pp.t/run.t

```diff
@@ -12,3 +12,8 @@ We also make sure that the dump file is not present
 
   $ dune_cmd exists profile.dump
   true
+
+This also works for reason code
+
+  $ dune describe pp src/main_re.re
+  ;;Util.log "Hello, world!"
```

Comment:
Since I was confused why this works (explanation: reason dialect is enabled by default, no need to enable it in `dune-project`) I think it would be nice to disable it and show that the dialect setting is being respected.

Reply:
yes. as discussed elsewhere, we'll make that more obvious in the docs.